### DISCUSSION TITLE
Does this config not work with Vite?

### DISCUSSION THREAD

**Discussion by beefchimi** [standard/eslint-config-standard-with-typescript#1565] [Q&A] [answered]
I've tried to get this setup on a minimal `vite` project here:
https://github.com/beefchimi/vite-eslint-flat

I've simply run the `npm create vite@latest` command and chose a `Vanilla > TypeScript` project.

There are not many commits on that demo repo, so it should be easy to see each small step I've taken to get things working.

Just trying something as simple as adding a `no-console` rule to the config _(or directly in the file as a comment)_ do not appear to work.

**Comment by mightyiam** [comment]
Sorry, could you please minimize that example a little more?

**Reply by beefchimi** [reply, marked as answer]
Hi!

So I think we have this _mostly sorted out_. A PR was opened by another contributor who made a few changes to get linting working: https://github.com/beefchimi/vite-eslint-flat/pull/1

I believe the real issue was that I had not specified the `files` paths correctly.

I am still struggling to get the `/dist` folder ignored... but that may be totally unrelated to `eslint-config-love`, so I believe we can mark this "Discussion" as resolved. 👍 

Thanks!

**Sidenote:**

The latest `v51` seems to be missing the `lib` folder:

![;pve](https://github.com/mightyiam/eslint-config-love/assets/643944/af72d489-1daf-4cea-8e8e-a6c2e303adfa)
